### ISSUE TITLE
Redis 7.4 support for new commands/options

### ISSUE THREAD

**Issue by kkmuffme** [phpredis/phpredis#2542]
https://github.com/redis/redis/releases/tag/7.4-rc1

New commands:
>HEXPIRE and HPEXPIRE set the remaining time to live for specific fields
HEXPIREAT and HPEXPIREAT set the expiration time to a UNIX timestamp for specific fields
HPERSIST removes the expiration for specific fields
HEXPIRETIME and HPEXPIRETIME get the expiration time for specific fields
HTTL and HPTTL get the remaining time to live for specific fields

+ various new flags as referenced on that link
+ various stuff I missed

Maybe we sticky this to create a list like how it was done for https://github.com/phpredis/phpredis/issues/2068

**Comment by Redtama** [comment]
Yes please! I could really use these commands in my PHP projects!

**Comment by michael-grunder** [comment]
We'll certainly implement them. As an aside, it's curious that there are nine different commands. Seems like they could have used `EX` or `PX` instead of duplicating each command.

I'm sure there was a reason.

Edit: If you need to use these commands now you can always do it via `rawCommand`.

**Comment by binaryfire** [comment]
I'd love to have these. I'm using `rawCommand` at the moment but it's not very clean. Any idea when they might be added? And thanks again for your work on this package.

**Comment by michael-grunder** [comment]
Going to close this since 6.3.0 is released which has every major feature of Redis 7.4.

IIRC there are a few administration level additions that we don't implement specific logic for but we can create another issue if someone asks for any of them
